libs/import/src/lib/view-extracted-elements/contract/utils.ts

```diff
@@ -149,24 +161,28 @@ export async function formatContract(
         };
         return centralOrgId.catalog;
       } else {
-        const sellerId = await getOrgId(value, orgService, orgNameCache);
-        if (!sellerId) return unknownEntityError('Licensor Organization');
+        let sellerId = await getOrgId(value, orgService, orgNameCache);
+        if (!sellerId) {
+          const seller = await orgService.getValue(value);
+          if (seller) sellerId = value;
+          else return unknownEntityError('Licensor Organization');
```

Comment:
Prefer the opposite : 
```suggestion
          if (!seller) return unknownEntityError('Licensor Organization');
          sellerId = value;
```
Avoid inlined `else` statement